test/perf/core.perf.js

```diff
@@ -394,13 +389,6 @@ describe.skip('Core Performance test', function () {
                         meta = core.getOwnJsonMeta(allMetaNodes[i]);
                     }
                     console.timeEnd('getOwnJsonMeta');
-
```

Comment:
it is completely unrelated.
we removed the 'meta' user as it was inefficient and didn't gave that much extra functionality.
and though are automated test-suite completely ignores this file, on my local machines it gave me headaches....
so I removed the portion from the performance tests which used the old implementation (there is no alternate on this level, as it was already a function covered by two implementations)